src/addons/plugin/property-proxy.js

```diff
@@ -67,6 +67,7 @@ class PropertyProxy extends Property {
       this.device.adapter.sendMsg(
         Constants.SET_PROPERTY, {
           deviceId: this.device.id,
+          adapterId: this.device.adapter.id,
```

Comment:
I think that this should really go in the AdapterProxy.sendMsg method, similar to how the Plugin.sendMsg injects the pluginId: https://github.com/mozilla-iot/gateway/blob/a5c90fbbe10594ea1cb6fbdda62c9afc4ac6d762/src/addons/plugin/plugin.js#L216